roles/pulibrary.psql/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: PostgreSQL | Make sure the CA certificates are available
+  apt:
+    pkg: ca-certificates
+    state: present
+
+- name: pgbouncer | Add PostgreSQL repository apt-key | apt
+  apt_key:
+    id: "{{ postgresql_apt_key_id }}"
+    url: "{{ postgresql_apt_key_url }}"
+    state: present
+
+- name: pgbouncer | Add PostgreSQL repository | apt
+  apt_repository:
+    repo: "{{ postgresql_apt_repository }}"
+    state: present
+
+- name: Postgresql | install postgresql server packages
+  apt:
+    name: '{{ item }}'
+    state: present
+    update_cache: true
+  loop:
+    - "{{ postgresql_packages }}"
+
+- name: Postgresql | install postgresql client packages
```

Comment:
Where and when does this get installed for user of this role?  Most the rails package installs these dependencies, may this is a moot point.